src/NexusMods.App.UI/WorkspaceSystem/WorkspaceController/WorkspaceController.cs

```diff
@@ -313,13 +313,61 @@ public void SetIcon(IconValue? icon, WorkspaceId workspaceId, PanelId panelId, P
         tabViewModel.Header.Icon = icon;
     }
 
-    public OpenPageBehavior GetDefaultOpenPageBehavior()
+    /// <inheritdoc/>
+    public OpenPageBehavior GetDefaultOpenPageBehavior(
+        PageData requestedPage,
+        NavigationInput input,
+        Optional<PageIdBundle> optionalCurrentPage)
     {
-        // TODO: Query user settings for default behaviors for:
-        // specific pages, specific interaction (mouse button modifiers), source workspace (if popped out or not)
+        const OpenPageBehaviorType fallback = OpenPageBehaviorType.NewTab;
```

Comment:
Shouldn't the fallback for opening from outside a workspace be Replace the first tab in first panel for now?
That is what design wanted at least

Reply:
This fallback is used in `GetValueOrDefault` calls when we try to look up a value in a Dictionary. If the Dictionary is populated correctly, then we will never use this fallback.

Reply:
I think what you're talking about is the default global setting behavior for the left menu.

Reply:
Right, yeah that should be Replace Tab for now (I don't particularly like it myself, but the idea is to try it and see)